client/src/components/App/Content/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -108,28 +111,44 @@ const SignUpForm: React.FC<Props> = ({ handleSaveUser }: Props) => {
                     </Grid>
                 </Grid>
 
-                <Grid container justify='flex-start' className={classes.formRow}>
-                    <Grid item xs={8}>
-                        <FormInput fieldName='שם מלא'>
-                            <Controller 
-                                name={`${SignUpFields.FULL_NAME}.${SignUpFields.FIRST_NAME}`}
+                { defaultValues ? 
+                    <Grid container justify='flex-start' className={classes.formRow}>
+                        <Grid item xs={8}>
+                            <FormInput fieldName='שם מלא'>
+                                <Controller 
+                                    name={SignUpFields.FULL_NAME}
+                                    control={methods.control}
+                                    render={(props) => (
+                                        GenericAlphabetTextField(props, 'שם מלא', '', classes.textField)
+                                    )}
+                                />
+                            </FormInput>
+                        </Grid>
+                    </Grid>
+                : 
+                    <Grid container justify='flex-start' className={classes.formRow}>
```

Comment:
What's the point in having different inputs for first and last name if we combine them later anyway? This should be discussed with Laly.

Reply:
Yes I know. for now you can close it